packages/logs/README.md

```diff
@@ -180,22 +180,37 @@ The results are the same when using NPM, CDN async or CDN sync:
 ```json
 {
   "status": "info",
-  "session_id": "1234",
+  "session_id": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
   "name": "buttonName",
   "id": 123,
   "message": "Button clicked",
+  "date": 1234567890000,
   "http": {
-    "url": "...",
-    "useragent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.130 Safari/537.36"
+    "useragent": "Mozilla/5.0 ...",
   },
-  "network": { "client": { "ip": "109.30.xx.xxx" } }
+  "view": {
+    "url": "https://...",
+    "referrer": "https://...",
+  },
+  "network": {
+    "client": {
+      "geoip": {...}
```

Comment:
FMU, geoip is provided by a custom processor on our log pipeline, it is not provided by the browser integration.
it could worth to double check though.

Reply:
I tried (in a new org), and it appears to be provided by the "Web Browser logs" integration pipeline automatically created when configuring the browser logs integration. Even if it can be disabled by the user, I think it's fine to keep mentioning this field, WDYT?

Reply:
ah, nice!
not sure why we have another geoip processor on our pipeline then 🤔
all good for the doc